src/models/custom.ts

```diff
@@ -0,0 +1,17 @@
+import { generateEntityId } from "@medusajs/utils"
```

Comment:
todo: Add in README in `src/models/README.md`

Reply:
there is already an example in readne, you want me to replace it or add a second example?

Reply:
Replace if this is more up to date

Reply:
done it 👍